app/views/foreman_ansible/job_templates/ansible_-_inventory.erb

```diff
@@ -0,0 +1,30 @@
+<%#
+name: Ansible Inventory
+snippet: false
```

Comment:
Ideally we should introduce description field to templates and put some description to this template saying this template has a special meaning and should not be renamed. This is not the first template that would benefit from it. This would mean a PR in Foreman core but would be a nice addition.

Reply:
Done: https://github.com/theforeman/foreman/pull/7051

Reply:
Code there looks good, but the use-case seems bit odd to me. @ares, how another editable field is going to stop me from making a mess, if I am a messy user? Can't we have lockable report_templates instead?

Reply:
if not, `Ansible Inventory` is suggestive enough for me to get an idea what that might be used for and why I shall not mess with it IMHO.